lib/common/components/SidebarPopover.js

```diff
@@ -36,6 +38,7 @@ export default class SidebarPopover extends React.Component<Props, State> {
 
   componentWillMount () {
     this._onResize()
+    this.props.onMount && this.props.onMount()
```

Comment:
Taking this further, we should actually be moving all mounting logic into the constructor of a component because [as of React 16.4 componentWillMount is deprecated and in React 17 componentWillMount will be removed entirely](https://medium.com/@sgobinda007/playing-with-component-lifecycle-methods-of-react-16-3-4c946c1e6982).  This also has the added benefit of allowing us to make much cleaner containers where the mapDispatchToProps would then be an object instead of the verbose function mapping it currently is in a lot of places.  I created issue https://github.com/catalogueglobal/datatools-ui/issues/246 to keep track of this.